packages/presets/near-operation-file/src/index.ts

```diff
@@ -268,7 +268,6 @@ export const preset: Types.OutputPreset<NearOperationFileConfig> = {
         config,
         schema: options.schema,
         schemaAst: schemaObject,
-        skipDocumentsValidation: true,
```

Comment:
This change is a bit too agressive, what we can do is to provide a more progressive configuration (cf: `skipDocumentsValidation` [has many options](https://www.graphql-code-generator.com/docs/config-reference/codegen-config#:~:text=skipDocumentsValidation%20%2D%20Allows%20to%20configure%20how%20to%20validate%20documents))

Suggested changes:

```js
skipDocumentsValidation: options.skipDocumentsValidation ? options.skipDocumentsValidation : { skipDuplicateValidation: true }
```

For this, `PresetFnArgs<Config, PluginConfig>` type will need to be updated accordingly to forward the `skipDocumentsValidation` top-level config property.